### ISSUE TITLE
Abnormal Result

### ISSUE THREAD

**Issue by fxcdl** [ofsoundof/GRL-Image-Restoration#19]
Hi, thank u for your code, I follow the README and run code with "torchx run......." on McMaster dataset successfully, but the result seems abnormal. 
![image](https://user-images.githubusercontent.com/50817018/234215652-d852edc3-b65f-46c3-88c9-f4c2b7a49820.png)
HQ is the same as LQ image

**Comment by ofsoundof** [comment]
Have you ever tried to compute the difference or PSNR? It's hard to tell whether the two images are the same by visual check.

**Comment by fxcdl** [comment]
Ah, thank u for your reply, I solved this problem. But when i tried to use GLR work on "Real-World Image SR", i got a new problem.
![image](https://user-images.githubusercontent.com/50817018/234275245-db33032e-706f-41f1-9fce-63cae11d8e77.png)
this is the command I used, and when the code runs to this "**print(OmegaConf.to_yaml(cfg, resolve=True))**"(line 187 in trainer.py), i got this error.
![image](https://user-images.githubusercontent.com/50817018/234276170-eeac199c-e893-4b3e-9dc7-6aae6fdb3e53.png)
Is there something wrong with my running command?
